app/controllers/account/members_controller.rb

```diff
@@ -26,6 +26,29 @@ def destroy
     end
   end
 
+  def greeting_new_member
```

Comment:
conditions can be simplified

redirect_to root_path, notice: 'Sorry, could not identify following link' unless @invitation

if  @invitation.expired?
  ...
else
  ...
end